lib/public-stats.js

```diff
@@ -274,3 +275,77 @@ const updateRetrievalTimings = async (pgClient, committees) => {
     JSON.stringify(rows)
   ])
 }
+
+/**
+ * @param {pg.Client} pgClient
+ * @param {Iterable<Committee>} committees
+ * @param {(minerId: string, cid: string) => (string[] | undefined)} findDealClients
+ */
+const updateDailyClientStats = async (pgClient, committees, findDealClients) => {
```

Comment:
```suggestion
const updateDailyClientRetrievalStats = async (pgClient, committees, findDealClients) => {
```

To be consistent with the line below and following